guides/vsf1_instance.md

```diff
@@ -0,0 +1,628 @@
+# Adding XDN to Vue Storefront Instance guide
+
+## 1. Install Vue Storefront
+
+Install VSF application using this guide: https://docs.vuestorefront.io/guide/installation/linux-mac.html
+
+<b>Note</b>: XDN requires Node version >= 12, so before the installation it's recommended to run:
+```bash
+nvm use 12
+```
+
+## 2. Prepare VSF files for XDN
+
+- In the new VSF project, go to `src/themes/default` (or any theme you're using) and remove `.git` folder from it to save that in Git VCS.
+- Go to `.gitignore` file and remove `config/local.json` line to keep it tracked.
+
+## 3. Install XDN
+
+- Install XDN packages: `yarn add -D @xdn/cli && yarn add @xdn/core @xdn/prefetch @xdn/devtools`
+- Next steps will cover the XDN key files you need to add. For more information about them see https://developer.moovweb.com/guides/starter
```

Comment:
```suggestion
```

I don't like sending SPA developers to the starter guide. It's a rabbit hole and I don't want to confuse them.  I think you can just continue on with the steps here.